packages/strapi/bin/strapi.js

```diff
@@ -126,6 +126,19 @@ program
   .description('generate a custom generator')
   .action(require('./strapi-generate'));
 
+// `$ strapi install`
+program
+  .command('install <plugin>')
```

Comment:
@odykyi this will be added in the Strapi v3 documentation. In the meantime, the explanations are in the description of the PR.